app/views/exercises/show.html.erb

```diff
@@ -61,3 +61,7 @@
   <%= render partial: 'layouts/modals/guide_corollary', locals: {guide: @guide} %>
   <%= render partial: 'layouts/modals/new_message', locals: {exercise: @exercise} if should_render_message_input?(@exercise) %>
 <% end if current_user? %>
+
+<script>
+    mumuki.setUpCurrentExp('<%= UserStats.exp_for(@current_user) %>');
```

Comment:
We should avoid those kind of scripts - because they are difficult to trace - and prefer a `mumuki.load` event on the proper module - I have actually performed similar refactors recently, in the kids PRs I think. 

Perhaps adding add global `mumuki.currentExp` could help

Reply:
I checked your PRs and found the script you removed, it's related to a modal. In this case, however, I need to do the initialization with `UserStats.exp_for(@current_user)`, and I can't do that when loading from the JS module itself - am I missing anything?

Reply:
Found this, I think this is what you meant?

https://github.com/mumuki/mumuki-laboratory/blob/master/app/assets/javascripts/mumuki_laboratory/application/exercise.js

I guess I could add a hidden `mu-current-exp` tag somewhere as with `mu-exercise-id` in that case

Reply:
That is right!